src/conferatur/normalization/__init__.py

```diff
@@ -0,0 +1,114 @@
+from collections import namedtuple
+import inspect
+from importlib import import_module
+from typing import Dict
+from conferatur.docblock import format_docs
+
+_normalizer_namespaces = (
+    "conferatur.normalization.core",
+    ""
+)
+
+NormalizerConfig = namedtuple('NormalizerConfig', ['name', 'cls', 'docs', 'optional_args', 'required_args'])
+
+
+def is_normalizer(cls):
+    return inspect.isclass(cls) and hasattr(cls, 'normalize')
```

Comment:
Is this a common pattern with Python? Or is it something that could be solved using subclasses?

Reply:
It is something that could be solved using subclassing, which would then check for a specific is_subclass_of, the normalizer base class would then be an abstract base class with abstract method normalize and we would test against this base class. Custom normalizers would then also need to subclass this abstract base class, making implementation slightly more complex.

Since python is duck typed, the method used above is quite common, it checks if the class has a "feature" method 'normalize', instead of is the class a normalizer class.